src/lib/constants.ts

```diff
@@ -12,6 +12,7 @@ export const LOCALES_CODES = i18nConfig.map((lang) => lang.code).sort()
 // Site urls
 export const SITE_URL = "https://ethereum.org" as const
 export const DISCORD_PATH = "/discord/" as const
+export const EDIT_CONTENT_URL = `https://github.com/ethereum/ethereum-org-website/tree/dev/` as const
```

Comment:
`as const` is not needed in a literal string. Also, I think we need to remove the last `/`. Otherwise, we get a double `//`
![image](https://github.com/ethereum/ethereum-org-fork/assets/468158/6f2dc343-f417-4ac0-ab17-f37c63ac5f94)

```suggestion
export const EDIT_CONTENT_URL = `https://github.com/ethereum/ethereum-org-website/tree/dev`
```

Reply:
Removed this when merging dev in